scribus/plugins/import/pdf/slaoutput.h

```diff
@@ -153,6 +153,73 @@ class AnoOutputDev : public OutputDev
 	QStringList *m_importedColors;
 };
 
+/* PDF TextBox Framework */
+/*
+* Holds all the dtails for each glyph in the text imported from the pdf file.
+*
+*/
+struct PdfGlyph {
+	QPointF position;    // Absolute glyph coords
+	double dx;  // X advance value
+	double dy;  // Y advance value
+	double rise;    // Text rise parameter
+	QString code;   // UTF-16 coded character but we only store and use UTF-8, the slternstive is const char * for utf8 so far as qt is concerned
+	bool is_space;
+};
+
+
+class TextRegionLine
+{
+public:
+	qreal maxHeight = -1;
+	qreal modeHeigth = -1;
+	qreal width = -1;
+	int glyphIndex = -1;
+	QPointF baseOrigin = QPointF(-1, -1);
+	std::vector<TextRegionLine> segments = std::vector<TextRegionLine>();
+
+};
+
+class TextRegion {
+public:
+	enum FRAMEWORKLINETESTS {

```

Comment:
if you don't plan to use other enums in this class, that's fine, otherwise i would use a class enum to give a scope to the enuzm values.

imo the enum name should be camelcase, starting with an uppercase F, not all uppercase. it's a type...

Reply:
ah, and the Scribus styles requires the `{` on tis own line...

Reply:
I've moved the enums into their own class now